### PR TITLE
[bitnami/wordpress] Update MariaDB to 14.x.x

### PR DESCRIPTION
**Pull request by Mauraza**
### Description of the change

Update MariaDB

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/wordpress/README.md [hunk 1]
```diff
@@ -513,6 +513,10 @@ To enable the new features, it is not possible to do it by upgrading an existing
 
 ## Upgrading
 
+### To 18.0.0
+
+This major release bumps the MariaDB version to 11.1. No major issues are expected during the upgrade.
```
Comment (juan131) [inline bitnami/wordpress/README.md:518]: Can we confirm so? Did you run manual tests to double-check it?

File: bitnami/wordpress/Chart.lock [hunk 2]
```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: memcached
   repository: oci://registry-1.docker.io/bitnamicharts
-  version: 6.6.2
+  version: 6.6.5
 - name: mariadb
   repository: oci://registry-1.docker.io/bitnamicharts
   version: 13.1.3
```
Comment (juan131) [inline bitnami/wordpress/Chart.lock:7]: This should've been bumped, right?

File: bitnami/wordpress/README.md (hunk 1 above)
Comment (Mauraza) [reply bitnami/wordpress/README.md:518]: Yes, I run locally, and everything going well.

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**